patches/server/0937-Fix-EndDragonFight-killed-statuses-should-be-false-f.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: elmital <54907162+elmital@users.noreply.github.com>
+Date: Thu, 8 Sep 2022 15:50:11 +0200
+Subject: [PATCH] Fix: EndDragonFight killed statuses should be false for newly
+ created worlds
+
+
+diff --git a/src/main/java/net/minecraft/world/level/dimension/end/EndDragonFight.java b/src/main/java/net/minecraft/world/level/dimension/end/EndDragonFight.java
+index f8d846345c1cc3c78f9ac14635b26f2affc77190..d380779f0fcdd00342b16b12f9d5e1aadb90fc46 100644
+--- a/src/main/java/net/minecraft/world/level/dimension/end/EndDragonFight.java
++++ b/src/main/java/net/minecraft/world/level/dimension/end/EndDragonFight.java
+@@ -91,7 +91,7 @@ public class EndDragonFight {
+     public EndDragonFight(ServerLevel world, long gatewaysSeed, CompoundTag nbt) {
+         // Paper start
+         this.needsStateScanning = world.paperConfig().entities.spawning.scanForLegacyEnderDragon;
+-        if (!this.needsStateScanning) this.dragonKilled = true;
++        this.dragonKilled = !this.needsStateScanning;
```

Comment:
I think this line isn't usefull since this field is already false at init stage. So the old line was enough.

Reply:
You right! I don't even remember why I've made this change. I reverted it in the last commit and rebase the PR in the same time.